packages/braintree-utils/src/utils/index.ts

```diff
@@ -1,3 +1,3 @@
-export { default as getValidBraintreeConnectStyles } from './get-valid-braintree-connect-styles';
+export { default as getValidBraintreeConnectStyles } from './get-valid-braintree-fastlane-styles';
```

Comment:
I don't really sure that getValidBraintreeConnectStyles is a right naming for the variable

Reply:
fixed